### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by grawk**
move 1.0 to master

### CODE REVIEW

File: app/index.js [hunk 1]
```diff
@@ -46,115 +46,12 @@ var NemoGenerator = yeoman.generators.Base.extend({
...
-      validate: function (key) {
-        if (key !== "") {
-          return true;
-        }
-        return "You need to provide an access key.";
-      },
-      when: function (answers) {
-        return (answers.sauceSetup === "Yes");
-      }
-    }];
```
Comment (nikulkarni) [inline app/index.js]: No sauce labs support or custom spec anymore? I feel sauce labs option add value to generator

Comment (grawk) [inline app/index.js]: I agree. It adds value. But I wanted to reduce the amount of questions and complexity in the main generator in order to give users the basic scaffold quickly. I think we should add sauce labs support as a sub generator. And also an option for perfecto mobile. Possibly other mobile testing options as well.

Comment (nikulkarni) [inline app/index.js]: yeah, someone during selenium meetup asked me this question if you can generate a sample test that runs on sauce labs using generator. I'll merge this for now and we can add a sub generator with documentation as a follow up PR

**Comment by nikulkarni** [comment]
Hmm.. It looks like I can't merge this PR. Did I lose my access to nemo* repos?

**Pull request merged**